routes/index.js

```diff
@@ -1,48 +1,52 @@
-const express = require('express');
+import express from 'express';
+
 const router = express.Router();
-const md5 = require('md5');
+
 
 /* GET welcome page. */
-router.get('/', (req, res, next)=> {
-  res.render('index', { title: 'PostIt Messaging App',login:'login' });
+router.get('/', (req, res) => {
+  res.render('index', { title: 'PostIt Messaging App', login: 'login' });
+});
+
+router.get('/example', (req, res) => {
+  res.status(200).send();
 });
 
 /* GET login page. */
-router.get('/login', (req, res, next)=> {
-  res.render('login', { title: 'PostIt Messaging App',login:'login' });
+router.get('/login', (req, res) => {
+  res.render('login', { title: 'PostIt Messaging App', login: 'login' });
 });
 
 
 /* GET register page. */
-router.get('/register', (req, res, next)=> {
-  res.render('register', { register:'Register' });
+router.get('/register', (req, res) => {
+  res.render('register', { register: 'Register' });
 });
 
 /* GET register page. */
-router.get('/password/reset', (req, res, next)=> {
-  res.render('email_reset', { reset:'Reset Password' });
+router.get('/password/reset', (req, res) => {
+  res.render('email_reset', { reset: 'Reset Password' });
 });
 
-/*route to create new group*/
-router.get('/new/group', (req, res, next)=> {
-  res.render('new_group', { group:'Create New Group' });
+/* route to create new group*/
+router.get('/new/group', (req, res) => {
+  res.render('new_group', { group: 'Create New Group' });
 });
 
-router.get('/groups', (req, res, next)=> {
+router.get('/groups', (req, res) => {
   res.render('message_board');
 });
 
-router.get('/new/message', (req, res, next)=> {
+router.get('/new/message', (req, res) => {
   res.render('new_message');
 });
 
-router.get('/home', (req, res, next)=> {
+router.get('/home', (req, res) => {
   res.render('index');
 });
 
 
-
-/*router.get('/api/group/:id/messages', function(req, res, next) {
+/* router.get('/api/group/:id/messages', function(req, res, next) {
   res.send(req.params);
 });*/
 
```

Comment:
it is better to use descriptive variables in your code base like request instead of req and response instead of res